File: MT.Model/Business/ProjectPresentProtected.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
	#region ProjectPresentProtected:EntityBase
	/// <summary>
	/// This object represents the properties and methods of a ProjectPresentProtected:EntityBase.
	/// </summary>
	public class ProjectPresentProtected:EntityBase
	{
        [Key]
		public Guid ProjectPresentProtectedID {get;set;}
		public Guid ProjectID{get;set;}
		public string DecisionNumber{get;set;}
		public DateTime DecisionDate{get;set;}
		public DateTime ProtectedDate{get;set;}
		public int Status{get;set;}
		public bool Inactive{get;set;}
		public int SortOrder{get;set;}
		
	
	}
	#endregion
}

File: BECore/app/model/ProjectPresentProtected.js
/// <summary>
/// Model EXTJS ProjectPresentProtected.
/// </summary>
Ext.define('QLDT.model.ProjectPresentProtected', {
    extend: 'Ext.data.Model',
    idProperty: 'ProjectID',
    idType: 'Guid',
    fields: [	
		{ name:"ProjectID", type:"string" },
		{ name:"DecisionNumber", type:"string" },
		{ name:"DecisionDate", type:"date" },
		{ name:"ProtectedDate", type:"date" },
		{ name:"Status", type:"int" },
		{ name:"Inactive", type:"boolean" },
		{ name:"SortOrder", type:"int" },
		{ name:"CreatedDate", type:"date" },
		{ name:"ModifiedDate", type:"date" },
		{ name:"IPAddress", type:"string" },
		{ name:"ModifiedBy", type:"string" },
		{ name:"CreatedBy", type:"string" },
     ]
});

